prov/util/src/util_attr.c

```diff
@@ -687,13 +719,24 @@ int ofi_check_rx_attr(const struct fi_provider *prov,
 		return -FI_ENODATA;
 	}
 
+	if (!rm_enabled &&
+	    user_attr->total_buffered_recv > prov_attr->total_buffered_recv) {
```

Comment:
"This field is supported for backwards compatibility purposes. It is a hint to the provider of the total available space that may be needed to buffer messages that are received for which there is no matching receive operation. The provider may adjust or ignore this value."  We should ignore this value, rather than throwing an error.  The field has essentially be deprecated except as a hint of buffering space that may be needed.

Reply:
Ok, I'm going to stay logging of a warning, but will just ignore the value

Reply:
I would just drop this if check and revert the API back.  The only reason the API changed was to add this check, which is only used to print a log message for a field that has been deprecated.